build/example.build.config

```diff
@@ -0,0 +1,9 @@
+name = KeyboardSize
+
+flex.sdk = /path/to/your/flexsdk/folder
+
+bin.ext = 
+
+ios.sdkversion = iphoneos
+
+android.sdk = /path/to/your/android-sdk/platforms/android-16
```

Comment:
it's an example. all of our example build configs looks like that. I'm using ver 20 of the sdk to compile.